binary.cabal

```diff
@@ -162,6 +162,7 @@ benchmark generics-bench
   build-depends:
     base >= 4.5.0.0 && < 5,
     bytestring >= 0.10.4,
+    generic-deriving >= 0.10,
```

Comment:
Note that the benchmark already depended on `generic-deriving` transitively. That was what caused one of the problems, as both `generic-deriving` and `GenericsBenchTypes` defined `instance Generic Version`.

Reply:
This is good and non-obvious info, please add that as a comment in the file.
And add how exactly we already depend on generic-deriving, which package depends on it?

Reply:
`criterion` depends on it via `aeson`.